concent_api/core/payments/bankster.py

```diff
@@ -1,12 +1,41 @@
+from typing import Optional
 from typing import Tuple
 
 from django.conf import settings
 
 from common.constants import ConcentUseCase
+from common.helpers import get_current_utc_timestamp
 from core.constants import ETHEREUM_ADDRESS_LENGTH
 from core.payments import service
 
 
+class ClaimPaymentInfo:
+
+    __slots__ = [
```

Comment:
could you please rearrange it, so it's aligned with arguments order of `__init__`

Reply:
:+1: done